### PR TITLE
Clearer instructions for popgen practical and improved lesson flow

### PR DESCRIPTION
**Pull request by MPriebe**

### CODE REVIEW

File: 2019/practicals/reference_genome/read-cleaning.md [hunk 1]
```diff
@@ -61,9 +61,8 @@ Sequencers aren't perfect. All kinds of things [can](http://genomecuration.githu
 
 Symlink the raw sequence files ('`/import/teaching/bio/data/reads.pe*.fastq.gz`) to a relevant input directory (e.g., `~/2019-09-xx-read_cleaning/input/`).
 
-Now move to a relevant results directory (e.g., `~/2019-09-xx-read_cleaning/results/`).
```
Comment (yeban) [inline 2019/practicals/reference_genome/read-cleaning.md]: We don't want to remove this. This is the first practical. We want to lay it out for them.

File: 2019/practicals/reference_genome/read-cleaning.md [hunk 2]
```diff
@@ -61,9 +61,8 @@ Sequencers aren't perfect. All kinds of things [can](http://genomecuration.githu
...
 
-In the results directory, run FastQC on the `reads.pe2` file. The command is given below but you will need to replace `REPLACE` with the path to a directory before running the command. The `--nogroup` option ensures that bases are not grouped together in many of the plots generated by FastQC. This makes it easier to interpret the output in many cases. The `--outdir` option is there to help you clearly separate input and output files. To learn more about these options run `fastqc --help` in the terminal.
```
Comment (yeban) [inline 2019/practicals/reference_genome/read-cleaning.md]: Same here, we want to keep 'In the results directory, ...'

Yannick will likely tell in lecture how to use the directory structure. But for the first time it's good to have a reminder.

File: 2019/practicals/reference_genome/read-cleaning.md (hunk 1 above)
Comment (MPriebe) [reply 2019/practicals/reference_genome/read-cleaning.md]: In the new project structure layout they want to be working in 2019-09-xx-read_cleaning/ from where they access the /input /tmp and /results directories. If they are in the results subdirectory they don't have access to /input or /tmp and will get confused.

File: 2019/practicals/reference_genome/read-cleaning.md (hunk 2 above)
Comment (MPriebe) [reply 2019/practicals/reference_genome/read-cleaning.md]: Again, with the new project structure there are no subdirectories within 2019-09-xx-read_cleaning/results so I don't think that is where they ought to be working

Comment (MPriebe) [reply 2019/practicals/reference_genome/read-cleaning.md]: If they are in the results directory we will need to change the fastqc command so input is ../input/xx and advise that output should be ../tmp ...i think this makes it confusing that we are asking them to work in /results

**Pull request merged**